iocore/net/UnixNet.cc

```diff
@@ -240,6 +241,8 @@ initialize_thread_for_net(EThread *thread)
 #else
   thread->ep->start(pd, thread->evpipe[0], nullptr, EVENTIO_READ);
 #endif
+
+  ++started_et_net_threads;
```

Comment:
Why have the polling continuations running, rather than here having something like
```
if (++started_et_net_threads == num_net_threads && num_accept_threads == 0) {
  start_HttpProxyServer();
}
```

Reply:
I remember it was because this still doesn't guarantee the information needed is there, so I have to schedule the continuation and stop it afterwards.

Reply:
Then you're getting lucky with the delay.

Reply:
Oh, and there's also the problem with the backdoor, need to differentiate the two some how.